modules/cluster-autoscaler-about.adoc

```diff
@@ -58,6 +58,8 @@ The cluster autoscaler takes pod priorities into account. The Pod Priority and P
 
 Pods with priority lower than the cutoff value do not cause the cluster to scale up or prevent the cluster from scaling down. No new nodes are added to run the pods, and nodes running these pods might be deleted to free resources.
 
+The cluster autoscaling is supported for the platforms that have machine API available on it.
```

Comment:
@elmiko Does this sentence look good? can we say "supported"?

Reply:
i think that's accurate to me

Reply:
Cool. Thanks